src/Tabs/Tab.tsx

```diff
@@ -0,0 +1,72 @@
+import { ClassNames, css } from '@emotion/core';
+import * as React from 'react';
+import { Tab } from 'react-tabs';
+import { useTheme } from '../theme';
+
+const StyledTab: React.FunctionComponent<{ disabled?: boolean }> & { tabsRole: string } = props => {
+  const theme = useTheme();
+  const styles = tabStyles(theme.tabs);
+  return (
+    <ClassNames>
+      {({ css: getClassName }) => (
+        <Tab
+          {...props}
+          className={getClassName(styles.tabStyle)}
+          disabledClassName={getClassName(styles.disabledTabStyle)}
+          selectedClassName={getClassName(styles.selectedTabStyle)}
+        >
+          {props.children}
+        </Tab>
+      )}
+    </ClassNames>
+  );
+};
+
+StyledTab.tabsRole = 'Tab';
+
+const tabStyles = (tabTheme: { fg: string; bg: string }) => {
+  return {
+    tabStyle: css({
+      display: 'inline-block',
+      border: '1px solid transparent',
+      borderBottom: 'none',
+      bottom: '-1px',
+      position: 'relative',
+      listStyle: 'none',
+      padding: '6px 12px',
+      cursor: 'pointer',
+      backgroundColor: tabTheme.bg,
+      color: tabTheme.fg,
+
+      '&:focus': {
+        boxShadow: '0 0 5px hsl(208, 99%, 50%)',
+        borderColor: 'hsl(208, 99%, 50%)',
+        outline: 'none',
+
+        '&:after': {
```

Comment:
`::after` is [preferred](https://developer.mozilla.org/en-US/docs/Web/CSS/::after) over `:after`

Reply:
Ah nice, I didn't know it!